src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -43,277 +48,1082 @@ public static void ClassCleanup()
+            //Verify Delete
+            page.StandardOperators.NumberPad.Input(3);
+            page.CalculatorApp.Header.SendKeys("3");
+            Assert.AreEqual("33", page.CalculatorResults.GetCalculatorResultText());
             page.StandardOperators.BackSpaceButton.Click();
-            Assert.AreEqual("-1,234", page.GetCalculatorResultText());
+            Assert.AreEqual("3", page.CalculatorResults.GetCalculatorResultText());
+            page.CalculatorApp.Header.SendKeys(Keys.Backspace);
+            Assert.AreEqual("0", page.CalculatorResults.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        [Priority(0)]
+        public void MixedInput_Operators_ClearEntryClear()
+        {
+            //Verify Clear Entery
+            page.StandardOperators.NumberPad.Input(3);
+            page.CalculatorApp.Header.SendKeys(Keys.Add);
+            page.CalculatorApp.Header.SendKeys("3");
+            page.CalculatorApp.EnsureCalculatorHasFocus();
+            page.CalculatorApp.Header.SendKeys(Keys.Delete);
+            Assert.AreEqual("0", page.CalculatorResults.GetCalculatorResultText());
+            Assert.AreEqual("3 +", page.CalculatorResults.GetCalculatorExpressionText());
+
+            page.StandardOperators.NumberPad.Input(9);
+            page.CalculatorApp.Header.SendKeys(Keys.Subtract);
+            page.CalculatorApp.Header.SendKeys("6");
+            page.StandardOperators.ClearEntryButton.Click();
+            Assert.AreEqual("0", page.CalculatorResults.GetCalculatorResultText());
+            Assert.AreEqual("3 + 9 Minus (", page.CalculatorResults.GetCalculatorExpressionText());
+
+            //Verify Clear
+            page.StandardOperators.NumberPad.Input(6);
+            page.CalculatorApp.Header.SendKeys(Keys.Subtract);
+            page.CalculatorApp.Header.SendKeys("9");
+            page.StandardOperators.ClearButton.Click();
+            page.CalculatorResults.CheckIfResultsExpressionIsClear();
         }
 
         // Issue #817: Prefixed multiple zeros
         [TestMethod]
-        public void Operator_Delete_Prefix_Zeros()
+        [Priority(2)]
+        public void MixedInput_Operators_DeletingDecimalDoesNotPrefixZeros()
         {
-            page.StandardOperators.NumberPad.Input(0.1); // To enter decimal point
+            page.StandardOperators.NumberPad.DecimalButton.Click(); // To enter decimal point
+            page.CalculatorApp.Header.SendKeys("1");
             page.StandardOperators.BackSpaceButton.Click();
             page.StandardOperators.BackSpaceButton.Click();
             page.StandardOperators.NumberPad.Input(0);
-            Assert.AreEqual("0", page.GetCalculatorResultText());
+            page.StandardOperators.NumberPad.Num0Button.Click();
+            page.StandardOperators.NumberPad.Input(0);
+            Assert.AreEqual("0", page.CalculatorResults.GetCalculatorResultText());
         }
 
+        #endregion
+
+        #region Standard Memory Tests
```

Comment:
Do History and Memory panels behave the same regardless of mode? If so, it might make more sense to move these to history and memory test suites instead of having them in the standard, scientific, etc test suites.

Reply:
Good idea.  To address this idea, I created CalculatorAppFunctionalTests.cs  This cs will contain history and memory tests, and any other future tests that are not specific to one mode.  The change will be in my next commit.